src/pudl/metadata/resources/eia860m.py

```diff
@@ -0,0 +1,56 @@
+"""Definitions of data tables primarily coming from EIA-860m."""
+from typing import Any
+
+RESOURCE_METADATA: dict[str, dict[str, Any]] = {
+    "core_eia861__monthly_generators_changelog": {
```

Comment:
Is this suppose to be yearly or monthly? I'm getting an error that `core_eia860m__yearly_generators_changelog_eia860m` doesn't exist in the metadata.

Reply:
doh yea i junked up the name of the transform asset. (and i smashed the materialize all assets after converting this to a ready for review and got two errors including this one) i will fix.